system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.cpp

```diff
@@ -52,7 +52,8 @@ LinuxProcessCpuMeasurementNode::LinuxProcessCpuMeasurementNode(
   const rclcpp::NodeOptions & options)
 : LinuxProcessCpuMeasurementNode{"linux_process_cpu_collector", options}
 {
-  Start();
+  configure();
```

Comment:
Is this a related change?

Reply:
Yes. In order for the ComposeableNode to start as a LifecycleNode, this is necessary. However, if someone feels strongly I can split into a separate PR, but didn't notice the state behavior until I performed manual testing with the launch file introduced here.